docs/guide/object-backgrounds.md

```diff
@@ -43,24 +43,28 @@ TexturedBackground {
 
 - `texture`: Defines the texture of the background.
 
+- `luminosity`: Is `Background.luminosity`.
+
+- `skybox`: Specifies if the background is used to define the textured skybox.
+
+- `reflections`: Specifies if the background is used in the material reflections.
+
+- `fallbackSkyboxColor`: Defines the background color in case the skybox is not defined.
```

Comment:
As mentioned in the previous comment, when reading these documentation I'm a little bit confused about the use of the terms "background", "skybox", and "texture".

I think that would be better to rename this field simply `backgroundColor`.

The description is ok like this only if no skybox is created in case the texture is not defined or disabled.
Otherwise instead of "the skybox is not defined" we should use "the `skybox` field is FALSE"